packages/icons/src/HarnessIcons.ts

```diff
@@ -2293,10 +2300,12 @@ const HarnessIcons: KVO<ElementType> = {
   up: Up,
   'upgrade-bolt': UpgradeBolt,
   'upload-box': UploadBox,
+  'user-groups': UserGroups,
   utility: Utility,
   valuesFIle: ValuesFIle,
   variable: Variable,
   'variables-blue': VariablesBlue,
+  'vector-stroke': VectorStroke,
```

Comment:
can we name this better including the word "arrow".. so that it's easier to find for others